### PR TITLE
feat: test on 2022 pipenv releases

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [x] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does
Test on 2022 pipenv releases & mark them as supported

### CODE REVIEW

File: packages/pipenv-pipfile/src/is-pipenv-supported-version.ts [hunk 1]
```diff
@@ -4,6 +4,14 @@ export function isPipenvSupportedVersion(
...
+    '2022.9.21',
+    '2022.8.31',
+    '2022.7.24',
+    '2022.6.7',
+    '2022.5.2',
+    '2022.4.21',
+    '2021.11.23',
     '2020.11.4',
     '2020.8.13',
     '2020.6.2',
```
Comment (admons) [inline packages/pipenv-pipfile/src/is-pipenv-supported-version.ts:17]: You didn't removed the versions you removed from being tested in the `pipenv_version` matrix on circle

File: packages/pipenv-pipfile/SUPPORTED.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Support Matrix
-This plugin has been tested with Python `latest patch` versions of `3.9`, `3.8`, `3.7`, `3.6`, `2.7` and the following `pipenv` versions:
+This plugin has been tested with Python `latest patch` versions of `3.10`, `3.9`, `3.8`, `3.7`, `3.6`, `2.7` and the following `pipenv` versions:
```
Comment (admons) [inline packages/pipenv-pipfile/SUPPORTED.md:2]: We removed 3.6 and 2.7

File: packages/pipenv-pipfile/src/is-pipenv-supported-version.ts (hunk 1 above)
Comment (lili2311) [reply packages/pipenv-pipfile/src/is-pipenv-supported-version.ts:17]: They are still supported, but I can't test them all as we have a jobs limit. Assuming those versions can't change and remain supported unless we drastically change the code

Comment (lili2311) [reply packages/pipenv-pipfile/src/is-pipenv-supported-version.ts:17]: Do you think it's best to remove them from this?

Comment (admons) [reply packages/pipenv-pipfile/src/is-pipenv-supported-version.ts:17]: Nope, you're right

**Review by admons — APPROVED** [review]

**Pull request merged**